ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using Task_1_7.Common;
+using System.IO;
+
+namespace ConsoleApp7
+{
+    public static class Program
+    {
+        public static FileReader fileReader = new FileReader();
+        public static void Main()
+        {
+            Console.WriteLine("Start...");
+            string filePath = Path.Combine(PathHelper.SolutionFolderPath, "Files", "Countries.txt");
+            Console.WriteLine("File Path is: " + filePath);
+            Dictionary<int, Country> dictionary = CreateDictionary(filePath);
+            if (dictionary != null)
+            {
+                dictionary = AddCountryToDictionary(dictionary);
+                dictionary = EditIsTelenorSupportedInDictionary(dictionary);
+                PrintDictionary(dictionary);
+                fileReader.PrintFile(ConvertDictionaryToList(dictionary), filePath);
+            }
+            Console.WriteLine("End...");
+            Console.ReadLine();
+        }
+
+        public static Dictionary<int, Country> CreateDictionary(string filePath)
+        {
+           if (fileReader.GetCountryListFromFile(filePath) != null)
+            {
+                int i = 0;
+                List<string> countryList = fileReader.GetCountryListFromFile(filePath);
+                Dictionary<int, Country> countries = new Dictionary<int, Country>();
+                string subline = ";";
+                int startIndex = 0;
+                foreach (string countryElement in countryList)
+                {
+                    int index = countryElement.IndexOf(subline);
+                    Country country = new Country
+                    {
+                        Name = countryElement.Substring(startIndex, index)
+                    };
+                    if (countryElement.Substring(index + 1).ToLower() == "true")
+                    {
+                        country.IsTelenorSupported = true;
+                    }
+                    else
+                    {
+                        country.IsTelenorSupported = false;
+                    }
+                    countries.Add(i + 1, country);
+                    i++;
+                }
+                return countries;
+            }
+            return null;
+        }
+
+        public static Dictionary<int, Country> AddCountryToDictionary(Dictionary<int, Country> dictionary)
```

Comment:
according to the actions that are done in the method, it should be called "AddUkraineToDictionary". But I would prefer if you pass the country value as a method parameter instead of hardcoding it